frontend/src/pages/question/questionBuilder/questionTemplate/subComponents/FillIdentifier.jsx

```diff
@@ -76,7 +51,7 @@ export default function FillIdentifier({
     // so we use a filter to remove those
     const newCuries = Object.values(normalizationResponse).filter((c) => c).map((c) => ({
       name: c.id.label || c.id.identifier,
-      type: ingestNodeType(c.type[0]),
+      type: c.type,
```

Comment:
I'm getting an issue looking up CURIEs in templates:

<img width="606" alt="Screen Shot 2021-01-16 at 1 04 33 PM" src="https://user-images.githubusercontent.com/1634866/104819513-f85a3000-57fb-11eb-9a55-428adb31f917.png">

Reply:
This is partly just me being lazy. But this will involve updating all the types in the query templates. I might just go ahead and update them to the new Reasoner standard format as well.